functions/src/games-list.ts

```diff
@@ -0,0 +1,18 @@
+import {https, logger} from "firebase-functions";
+import * as admin from "firebase-admin";
+
+interface Game {
+  id: string;
+  name: string;
+  platform: string;
+  score: number;
+}
+
+const GAMES_LIST_REF = "/1a8pzsOq0j1VNEjq_MaQKUJy2-tN1Mi4XiIjn6iczYHI/Лист1";
```

Comment:
Maybe to store this ref in props file and don't store it as a part of the code?
It seems to me to be non-secured

Reply:
That's not going to the js bundle and this piece of data actually does not look as a secure one